cfgen/cfgen

```diff
@@ -328,10 +329,14 @@ def validate_pools_desc(pools_desc: List[Dict[str, Any]]) -> None:
         t = pool_type(pool)
         if t is PoolT.sns:
             d = pool.get('allowed_failures')
-            assert d is None or d['disk'] <= pool['parity_units'], """\
+            spare_units = pool.get('spare_units')
+            assert spare_units is None\
+                or spare_units == 0\
+                or spare_units == pool['parity_units']
+            assert d is None or d['disk'] <= pool['spare_units'], """\
```

Comment:
No, still it must be allowed_failures<= pool['parity_units'] because even if spare_units = 0 and parity_units > 0, we can tolerate failures = parity_units.

Reply:
Done. Thanks!